index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/styles.css">
+  <title>Portfolio</title>
+</head>
+<body>
+  <header class="header-container">
+    <div class="header-item header-item1"><time id="time">9:41</time></div>
+    <div class="header-item header-item2"><img id="signal" src="src/signal.png" alt="signal"></div>
+    <div class="header-item header-item3"><p id="name">Anand Pathak</p></div>
+    <div class="header-item header-item4"><img id="tab" src="src/tab.png" alt="tab"></div>
+  </header>
+
+  <section class="intro-section-container">
+    <p class="intro-section-item" id="intro-text1">Hey There. I'm Anand</p>
+    <p class="intro-section-item" id="intro-text2">I’am a Software Developer</p>
```

Comment:
I suggest you use the correct tag for this subtitle
Subtitles uses a header tag `hx (examples: <h1>,<h2>,<h3>, etc...)`.
* This way will be semantically correct and the screen readers and other technologies will thank you.